firebase.py

```diff
@@ -3,21 +3,43 @@
 '''
 
 '''dummy imports please remove them if found'''
-
-import pickle as pkl
+from unicodedata import name
+import firebase_admin
+from firebase_admin import storage
+import os
 ''''''
 
 
-
-
-
-
-
 # Download models from firebase : Returns list of models
 def downloadModels(project_id):
-    pass
+    ds = storage.bucket()
+    file_names = list()
+    L = len(project_id)
+    for b in ds.list_blobs(): 
+        file_names.append(b.name)
+    print(file_names , L )
+    
+    for file_path in file_names :
+        file_dir = file_path[0:L]
+        print(file_dir , file_path[L+1:])
+        if(file_dir == project_id and len(file_path) > L+1):
+            print("matched",file_path[L+1:])
+            bob = ds.blob(file_path)
+            bob.download_to_filename("model-files/local/"+ file_path[L+1:])
 
 
 # Upload model to firebase : Returns response object with status property
-def uploadModel(model, project_id):
-    pass
\ No newline at end of file
+def uploadModel(project_id):

```

Comment:
Why only project_id is given as input? What should the aggregation function do before calling this method?

Reply:
The Model will be Uploaded into model-files/globalModel.pkl before the uploadModel.